.github/workflows/dockerx.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Push Headless Images
+'on':
+  schedule:
+    - cron: 0 1 * * *
+  push:
+    branches-ignore:
+      - none
+  workflow_dispatch: null
+jobs:
+  build:
+    runs-on: self-hosted
```

Comment:
I think we can run these directly on github without s6.hive

This is an open source project